modules/gitops-openshift-go-common-terms.adoc

```diff
@@ -0,0 +1,137 @@
++
+Within the `openshift-gitops` namespace, Argo CD maintains a set of following Kubernetes resources, which define the continuous deployment between the source Git repository and destination clusters:
+
+* Argo CD `Application` custom resources (CRs)
+* `ConfigMap` API objects
+* `Secret` objects representing the {gitops-shortname} repository credentials and cluster credentials for deployment targets
+
+Manifest::
+In the {gitops-shortname} context, a manifest is a YAML representation of Kubernetes resources defined within a {gitops-shortname} repository, with the intent to deploy those resources to a target Kubernetes cluster. An example of a manifest would be the YAML representation of resources such as `Deployment`, `ConfigMap`, or `Secret`.
+
+Workload::
+Any process usually defined within resources such as `Deployment`, `StatefulSet`, `ReplicaSet`, `Job`, or `Pod`, running within a container. Examples include a Spring Boot application, a NodeJS Express application, or a Ruby on Rails application.
+
+Application::
+A group of Kubernetes resources as defined by a manifest. This is a Custom Resource Definition (CRD). An `Application` CRD is a resource object representing a deployed application instance in an environment.
+
+Argo CD application::
+An Argo CD application tracks the continuous deployment of individual Kubernetes resources from the {gitops-shortname} repository, where the resources are defined as manifests, to a target Kubernetes cluster.
+
+Declarative setup::
+A declarative description of the infrastructure you need in your specified environment, for system and application setup or configuration. You can specify this description in a YAML configuration file in the Git repository. The declarative setup contains an automated process to make your environment and infrastucture match the described state. Examples include defining Argo CD applications, projects and settings declaratively using YAML manifests.
+
+Live state::
+The live state of application resources on a target cluster.
+
+Target state::
+The desired state of application resources, as represented by files in a Git repository.
+
+Sync::
+The process of synchronizing the live state of an application's cluster resources with the target state defined within Git to ensure consistency, by using the Argo CD UI. Examples include syncing an application by applying changes to a cluster.
+
+Sync status::
+The status of an application that indicates whether the live state matches the target state.
+
+Refresh::
+The process to compare the latest code in Git with the live state and figure out the difference, by using the Argo CD UI. For example, in the Argo CD UI, when you click *Refresh*, Argo CD connects to an application's target Git repository, retrieves the content, and then generates manifests from that content. Argo CD then compares that target state against the live cluster state.
```

Comment:
Suggestion: Discard it if you feel otherwise: 

to compare the latest code in Git -> to compare the latest code in the Git repository